docs/tasks/manage-stateful-set/upgrade-pet-set-to-stateful-set.md

```diff
@@ -0,0 +1,163 @@
+---
+assignees:
+- bprashanth
+- enisoc
+- erictune
+- foxish
+- janetkuo
+- kow3ns
+- smarterclayton
+
+---
+
+{% capture overview %}
+This page shows how to upgrade from PetSets (Kubernetes version 1.3 or 1.4) to *StatefulSets* (Kubernetes version 1.5 or later).
+{% endcapture %}
+
+{% capture prerequisites %}
+
+* If you don't have PetSets in your current cluster, or you don't plan to upgrade
+  your master to Kubernetes 1.5 or later, you can skip this task. 
+
+{% endcapture %}
+
+{% capture steps %}
+
+### Differences between alpha PetSets and beta StatefulSets
+
+PetSet was introduced as an alpha resource in Kubernetes release 1.3, and was renamed to StatefulSet as a beta resource in 1.5. 
+Here are some notable changes:
+
+* **StatefulSet is the new PetSet**: PetSet is no longer available in Kubernetes release 1.5 or later. It becomes beta StatefulSet. To know why the name is changed, see this [discussion thread](https://github.com/kubernetes/kubernetes/issues/27430).
+* **StatefulSet guards against split brain**: StatefulSets guarantee at most one Pod for a given ordinal index can be running anywhere in a cluster, to guard against split brain scenarios with distributed applications. *TODO: Link to doc about fencing*
+* **Flipped debug annotation behavior**: The default value of the debug annotation (`pod.alpha.kubernetes.io/initialized`) is now `true`. The absence of this annotation will pause PetSet operations, but will NOT pause StatefulSet operations. In most cases, you no longer need this annotation in your StatefulSet manifests. 
+
+
+### Upgrading from PetSets to StatefulSets
+
+Note that these steps need to be operated in the specified order. You **should
+NOT upgrade your Kubernetes master, nodes, or `kubectl` to Kubernetes version
+1.5 or later**, until told to do so.
+
+#### Find all PetSets and their manifests 
+
+First, find all existing PetSets in your cluster:
+
+```shell
+kubectl get petsets --all-namespaces
+```
+
+If you don't find any existing PetSets, you can safely upgrade your cluster to
+Kubernetes version 1.5 or later.
+
+If you find existing PetSets and you have all their manifests at hand, you can continue to the next step to prepare StatefulSet manifests. 
+
+Otherwise, you need to save their manifests so that you can recreate them as StatefulSets later. 
+Here's an example command for you to save all existing PetSets as one file. 
+
+```shell
+# Save all existing PetSets in all namespaces into a single file. Only needed when you don't have their manifests at hand. 
+kubectl get petsets --all-namespaces -o yaml > all-petsets.yaml
+```
+
+#### Prepare StatefulSet manifests 
+
+Now, for every PetSet manifest you have, prepare a corresponding StatefulSet manifest with the following steps: 
```

Comment:
Suggestion: Strike "with the following steps."

... prepare a corresponding StatefulSet manifest: